src/ts/components/settingspanelpage.ts

```diff
@@ -73,10 +73,12 @@ export class SettingsPanelPage extends Container<ContainerConfig> {
   }
 
   onActiveEvent() {
-    this.settingsPanelPageEvents.onActive.dispatch(this);
+    const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent);
```

Comment:
Please integrate this into https://github.com/bitmovin/bitmovin-player-ui/blob/master/src/ts/browserutils.ts